docs/docs/tutorials/deployment.md

```diff
@@ -17,7 +17,17 @@ Get started deploying your LangGraph applications locally or on the cloud with
 
 ## Deployment Options
 
-- [Self-Hosted Lite](../concepts/self_hosted.md): A free (up to 1 million nodes executed per year), limited version of LangGraph Platform that you can run locally or in a self-hosted manner
-- [Cloud SaaS](../concepts/langgraph_cloud.md): Hosted as part of LangSmith.
-- [Bring Your Own Cloud](../concepts/bring_your_own_cloud.md): We manage the infrastructure, so you don't have to, but the infrastructure all runs within your cloud.
-- [Self-Hosted Enterprise](../concepts/self_hosted.md): Completely managed by you.
+- [Cloud SaaS](../concepts/langgraph_cloud.md): Connect to your GitHub repositories and deploy LangGraph Servers to LangChain's cloud. We manage everything.
```

Comment:
MAJOR: Could we add any pricing/plan information to this table? This has been a point of confusion for many users.

Reply:
![image](https://github.com/user-attachments/assets/e528aa6e-0529-422d-b690-b11940127210)

I think this is the best I can do right now. `Pricing` links to the public pricing page for LangGraph Platform, which is also confusing and needs a refactor. But I agree, it's a good row to have and probably top of mind for everyone. Thoughts?